src/ECER.Managers.Admin.Contract/ECER.Managers.Admin.Contract.csproj

```diff
@@ -2,4 +2,7 @@
   <ItemGroup>
     <PackageReference Include="MediatR.Contracts" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\ECER.Resources.Accounts\ECER.Resources.Accounts.csproj" />
```

Comment:
a contract should never reference another project, it's a contract and should be independent from any dependencies